GDJS/Runtime/runtimeobject.js

```diff
@@ -34,27 +41,64 @@
  * @param {ObjectData} objectData The initial properties of the object.
  */
 gdjs.RuntimeObject = function(runtimeScene, objectData) {
+    /** @type {string} */
```

Comment:
Can you add protected to all the `this.xxx` members of RuntimeObject? None of these should appear in the documentation as public because they are internal state of the object and should not be modified from the outside.

Reply:
See if you can generate the documentation for GDJS:

https://github.com/4ian/GDevelop/blob/7c9e6ee6f135f0b526f89f9ebec903868bab0249/scripts/GenerateAllDocs.sh#L19-L21

and look at what is generated for RuntimeObject.

Reply:
shouldn't protected vars begin with an underscore?

Reply:
In theory yes, in practice they have been named like this before such convention was properly used in the codebase. Ideally we could rename them, but this is a more risky refactoring to be done.